ee/ucp/admin/configure/enable-audit-logging.md

```diff
@@ -1,99 +1,95 @@
 ---
 title: Enable audit logging on UCP
-description: Learn how to enable audit logging of all activity in UCP
+description: Learn how to enable audit logging of all activity in UCP.
 keywords: logs, ucp, swarm, kubernetes, audits
 redirect_from: /ee/ucp/admin/configure/create-audit-logs/
 ---
 
 >{% include enterprise_label_shortform.md %}
 
 Audit logs are a chronological record of security-relevant activities by 
-individual users, administrators or software components that have affected the 
+individual users, administrators, or software components that have affected the 
 system. They are focused on external user/agent actions and security rather than 
 understanding state or events of the system itself.
 
 Audit logs capture all HTTP actions (GET, PUT, POST, PATCH, DELETE) to all UCP 
-API, Swarm API and Kubernetes API endpoints that are invoked (except for the 
-ignored list) and sent to Docker Engine via stdout. Creating audit logs is a UCP 
-component that integrates with Swarm, Kubernetes, and UCP APIs.
+API, Swarm API, and Kubernetes API endpoints that are invoked (except for the 
+ignored list) and sent to Docker Engine through standard output (stdout). Creating audit logs is a UCP component that integrates with Swarm, Kubernetes, and UCP APIs.
```

Comment:
I suggest putting the last sentence into parenthesis at the end of the previous sentence.